src/examples/rest/src/backend/database.ts

```diff
@@ -9,7 +9,7 @@ export const myConnection = {
 		driver: MySqlDriver,
 		dbName: 'todo_app',
 		user: 'root',
-		password: 'password',
+		password: '',
```

Comment:
Whilst we are here can we replace these with the env vars?

in the .env example:

MYSQL_USERNAME="root"
MYSQL_PASSWORD="password"